src/streamlink/stream/dash.py

```diff
@@ -196,6 +196,22 @@ def parse_manifest(cls, session, url, **args):
         if not audio:
             audio = [None]
 
+        locale_lang = session.localization.language.alpha2
```

Comment:
> alpha2

there are some `alpha3` codes --> `rus`

https://github.com/streamlink/streamlink/blob/c1489782cf2644e9194c4d2fdcc3f6198e49a538/tests/resources/dash/test_4.mpd#L25


---

there are also some other, but thats more of an `--dash-audio-select` command

```
Available languages for DASH audio streams:
en_surround, no-voices_stereo, en_stereo (using: en_stereo)
```
https://github.com/streamlink/streamlink/blob/c1489782cf2644e9194c4d2fdcc3f6198e49a538/tests/resources/dash/test_2.mpd#L3-L15

Reply:
We should add `--dash-audio-select` too :)